### PR TITLE
enable to append params for FCM priority, which is compatible to GCM

### PR DESCRIPTION
**Pull request by t-kurimura**
```
$ go test -run 'TestComposeMessageAPNS'
PASS
ok  	github.com/t-kurimura/aws-sdk-go-wrapper/sns	1.691s
```

```
$ go test -run 'TestPublish'
PASS
ok  	github.com/t-kurimura/aws-sdk-go-wrapper/sns	1.715s
```

c.f. https://firebase.google.com/docs/cloud-messaging/concept-options#header

### CODE REVIEW

**Comment by evalphobia** [comment]
@t-kurimura Thank you for your PR. The feature would be helpful.
IMO, I don't want to add arguments to `Publish()` for device specific features.

- (a) Use more flexible arguments of other device specific features for the future extension.
- (b) Set `"android" : "high"` as default parameter for GCM.  (If it does not have obvious disadvantages)
    - [APNs use maximum priority(10) as a default value.](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/sending_notification_requests_to_apns)

Would you think above points?

File: sns/helper.go [hunk 1]
```diff
@@ -21,10 +21,25 @@ func composeMessageGCM(msg string, opt map[string]interface{}) (payload string,
...
 	b, err := json.Marshal(message)
 	return string(b), err
 }
 
+// set Android FCM priority, which is compatible to GCM
+func appendPriority(msgVal map[string]interface{}, isHighPriority bool) map[string]interface{}  {
+
+	var priority string
+	if isHighPriority {
+		priority = "high"
```
Comment (evalphobia) [inline sns/helper.go]: [nits]
It's good to use const for predefined strings.

```
const gcmPriorityHigh = "high"
```

**Comment by t-kurimura** [comment]
@evalphobia Thanks for your review.
I understood your intention, I chose (b)  option. i"d like you to confirm again.

**Comment by evalphobia** [comment]
@t-kurimura  Thanks! 🍔 

Note: https://github.com/evalphobia/aws-sdk-go-wrapper/releases/tag/v1.4.0

**Pull request merged**